src/main/java/com/alinab/taskSevenMySQLIntegration/controllers/OrdersController.java

```diff
@@ -0,0 +1,116 @@
+package com.alinab.taskSevenMySQLIntegration.controllers;
+
+import com.alinab.taskSevenMySQLIntegration.models.Order;
+import com.alinab.taskSevenMySQLIntegration.models.Product;
+import com.alinab.taskSevenMySQLIntegration.service.OrdersService;
+import com.alinab.taskSevenMySQLIntegration.service.ProductsService;
+import com.alinab.taskSevenMySQLIntegration.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping("/orders")
+public class OrdersController {
+
+    private final OrdersService ordersService;
+    private final ProductsService productsService;
+    private final UsersService usersService;
+
+    @Autowired
+    public OrdersController(OrdersService ordersService, ProductsService productsService, UsersService usersService) {
+        this.ordersService = ordersService;
+        this.productsService = productsService;
+        this.usersService = usersService;
+    }
+
+    @GetMapping("/showOrder/{id}")
```

Comment:
no verb in path, just GET /orders/{id}
https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/
https://www.partech.nl/nl/publicaties/2020/07/9-trending-best-practices-for-rest-api-development